packages/components/psammead-social-embed/src/Notice/index.jsx

```diff
@@ -36,8 +32,12 @@ const Wrapper = styled.div`
     ${({ service }) => getSansBold(service)}
     text-decoration: none;
 
-    > span {
-      ${visuallyHiddenStyle}
+    &:visited {
+      color: ${C_METAL};
+    }
+
+    &:hover {
```

Comment:
focus too please?

Reply:
Focus comes by default.

Reply:
the default outline yes, but we would also like text underline too please (as the default outline doesn't always pass contrast depending on the browser)